File: 1-easy/problems/3718-smallest-missing-multiple-of-k/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def missingMultiple(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        lookup = set(nums)
        for i in range(1, len(lookup)+1):
            if i*k not in lookup:
                return i*k
        return (len(lookup)+1)*k

File: 1-easy/problems/3718-smallest-missing-multiple-of-k/solution.ts
function missingMultiple(nums: any, k: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :type k: int
  :rtype: int
  """
  lookup = set(nums)
  for (i in Array.from({length: 1, (lookup}, (_, i) => i).length+1)) {
      if (i*k not in lookup) {
          return i*k;
  return ((lookup).length+1)*k;
}

export { missingMultiple };
